shared/controller-api/src/main/java/io/pravega/shared/controller/event/AutoScaleEvent.java

```diff
@@ -23,7 +23,7 @@
     private final String stream;
     private final long segmentId;
     private final byte direction;
-    private final long timestamp;
+    private final long timestamp; // This field will act as requestId
```

Comment:
This is odd and hackish. Let's not do this and think of a way of not overloading fields.

Reply:
Now `ScaleOpEvent` and `AutoScaleEvent` contain a `requestId` field for tracing.